ratelimits/testdata/working_override_13371338.yml

```diff
@@ -12,3 +12,10 @@
     ids:
       - id: 13371338
         comment: Used to test the TransactionBuilder
+- FailedAuthorizationsForPausingPerDomainPerAccount:
+    burst: 1337
+    count: 1337
+    period: 5m
```

Comment:
nit: let's us semi-realistic values here and in the other test file. At least, let's make sure that the count is low and the period is highish, so this doesn't just look like all of our other limits.

Reply:
Let's do something like burst 1337, count 1, period 24h